test/wpt/runner/worker.mjs

```diff
@@ -89,6 +90,11 @@ Object.defineProperties(globalThis, {
   CacheStorage: {
     ...globalPropertyDescriptors,
     value: CacheStorage
+  },
+  // TODO: remove once node 18 is dropped
+  crypto: {
+    ...globalPropertyDescriptors,
+    value: webcrypto
```

Comment:
While this may satisfy the WPT requirements it's not a proper WebCryptoAPI polyfill. Couldn't you just set NODE_OPTIONS to `--experimental-global-webcrypto` instead?

Reply:
I have vague recollections of having issues with windows when adding options

Reply:
Right, proceed as you see best fit then?

Reply:
I conditionally add it